bower.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "nvd3",
-  "version": "1.7.1",
```

Comment:
They don't, but the gruntfile reads in the bower json to get the version, so the version is stored only in a single place :)